### PR TITLE
Add util function for hodl invoice detection

### PR DESCRIPTION
**Pull request by benthecarman**
No reason to keep the disction between test and non test for the list, just keeping the `ONE_KEY` in there because no one will ever have it anyways

### CODE REVIEW

File: mutiny-wasm/src/lib.rs [hunk 1]
```diff
@@ -1705,6 +1705,14 @@ impl MutinyWallet {
         let npub = parse_npub_or_nip05(&npub).await?;
         Ok(npub.to_bech32().expect("bech32"))
     }
+
+    /// If the invoice is from a node that gives hodl invoices
+    #[wasm_bindgen]
+    pub async fn is_potential_hodl_invoice(invoice: String) -> Result<bool, MutinyJsError> {
```
Comment (TonyGiorgio) [inline mutiny-wasm/src/lib.rs:1711]: Why is this needed? Don't we already warn users when paying zeus that it is a hodl invoice? Something is already doing that logic somewhere based on the list of pubkeys.

Comment (benthecarman) [reply mutiny-wasm/src/lib.rs:1711]: In the front end we only warn for lightning address by checking for @zuespay.com

**Review by TonyGiorgio — APPROVED** [review]
